components/application-mgt/org.wso2.carbon.identity.application.mgt/src/main/java/org/wso2/carbon/identity/application/mgt/dao/impl/ApplicationMgtDBQueries.java

```diff
@@ -266,6 +266,7 @@ public class ApplicationMgtDBQueries {
     public static final String REMOVE_UM_ROLE_PERMISSION = "DELETE FROM UM_ROLE_PERMISSION WHERE UM_PERMISSION_ID = ?";
 
     // DELETE query - Oauth
+    @Deprecated
```

Comment:
what's the new query to be used here?

Reply:
I don't think we can add a deprecation note as this is a variable. Added a comment

Reply:
Fixed